.github/workflows/income-limits-data-sync.yml

```diff
@@ -2,7 +2,7 @@ name: Income Limits Data Sync
 on:
   workflow_dispatch: null
   schedule:
-    - cron: "35 0 1 * *"
+    - cron: "35 0 28 * *"
```

Comment:
Changes the frequency of the GHA to fire _before_ the scheduled Sidekiq scripts.